playbooks/upload_file_to_cloud_storage/README.md

```diff
@@ -0,0 +1,53 @@
+# cloud.gcp_ops.upload_file_to_cloud_storage
+
+A playbook to upload a file from a remote host to GCP Cloud Storage.
+
+## Requirements
+
+GCP user account with at minimum the storage.objectCreator IAM role or equivalent permissions.
+
+## Playbook Variables
+
+* **upload_file_to_cloud_storage_bucket_name**: The name of the GCP storage bucket to upload the file to. **Required**
+* **upload_file_to_cloud_storage_file_path**: Path to file to upload. **Required**
+* **upload_file_to_cloud_storage_host_name**: Name of the remote host or group with file to upload. Default: `"all"`.
+* **upload_file_to_cloud_storage_object_name**: The name to give the object in cloud storage. If not provided, defaults to the base file name (without path) of `upload_file_to_cloud_storage_file_path`.
+
+## Dependencies
+
+* Role: [gcp_setup_credentials](https://github.com/redhat-cop/cloud.gcp_ops/blob/main/roles/gcp_setup_credentials/README.md)
```

Comment:
You should remove this because we are not using anymore the gcp_setup_credentials role.

Reply:
Done